politeiawww/config.go

```diff
@@ -41,12 +41,13 @@ const (
 )
 
 var (
-	defaultHTTPSKeyFile  = filepath.Join(sharedconfig.DefaultHomeDir, "https.key")
-	defaultHTTPSCertFile = filepath.Join(sharedconfig.DefaultHomeDir, "https.cert")
-	defaultRPCCertFile   = filepath.Join(sharedconfig.DefaultHomeDir, "rpc.cert")
-	defaultCookieKeyFile = filepath.Join(sharedconfig.DefaultHomeDir, "cookie.key")
-	defaultLogDir        = filepath.Join(sharedconfig.DefaultHomeDir, defaultLogDirname)
-	defaultPaywallAmount = uint64(0)
+	defaultHTTPSKeyFile             = filepath.Join(sharedconfig.DefaultHomeDir, "https.key")
+	defaultHTTPSCertFile            = filepath.Join(sharedconfig.DefaultHomeDir, "https.cert")
+	defaultRPCCertFile              = filepath.Join(sharedconfig.DefaultHomeDir, "rpc.cert")
+	defaultCookieKeyFile            = filepath.Join(sharedconfig.DefaultHomeDir, "cookie.key")
+	defaultLogDir                   = filepath.Join(sharedconfig.DefaultHomeDir, defaultLogDirname)
+	defaultPaywallAmount            = uint64(1)
```

Comment:
This is a silly default. Either make this something sensical or make it 0 and complain at startup. We should not allow anything < dust.